parseutils.c

```diff
@@ -560,7 +565,17 @@ get_int64_from_file(char *ftr)
 		result = PG_INT64_MAX;
 	else
 	{
+        #if PG_VERSION_NUM < 150000
 		success = scanint8(rawstr, true, &result);
+		#endif
+		#if PG_VERSION_NUM >= 150000
+		errno = 0;
+		result = strtoi64(rawstr, &endptr, 10);
+		if (errno == 0)
+		{
+			success = true;
+		}
+		#endif
```

Comment:
I'd probably do something like:
```suggestion
#else
		result = strtoi64(rawstr, &endptr, 10);
		success = !errno;
#endif
```

Reply:
Done